### PR TITLE
Added a header for May 2020

### PR DESCRIPTION
**Pull request by adognameddog**
What it says on the tin

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -372,7 +372,7 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
-#### Latest
+#### May 2020
```
Comment (luisaterrabrito) [inline apiary.apib:375]: Can you add a new latest?

**Review by dietrichm — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (adognameddog) [reply apiary.apib:375]: I could, but there would be nothing in there?

**Review by luisaterrabrito — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (luisaterrabrito) [reply apiary.apib:375]: Yeah let's add it again when there are new changes

**Review by ketlane — APPROVED** [review]

**Comment by dietrichm** [comment]
✅ Checked

**Pull request merged**